example/lib/main.dart

```diff
@@ -18,7 +16,7 @@ const String testFeature = 'show_title_logo';
 
 void setupPrefs() {
   getIt.registerSingleton<FlagsmithClient>(FlagsmithClient(
-      apiKey: 'EBnVjhp7xvkT5oTLq4q7Ny',
+      apiKey: 'CoJErJUXmihfMDVwTzBff4',
```

Comment:
So this is replacing the existing API key with your own? I suppose this is fine so long as the environment isn't adjusted / removed.

Reply:
yep i lost my account access after putting this client under your arms because of switching from standard to organisation profile on github :(